test/test_ion_stream.cpp

```diff
@@ -99,7 +99,7 @@ TEST(IonStream, ContinuesOverPageBoundary) {
     ION_ASSERT_OK(ion_reader_close(reader));
     ION_ASSERT_OK(ion_stream_close(stream));
 
-    ASSERT_EQ(2, context.number_of_handler_invocations);
+    ASSERT_EQ(3, context.number_of_handler_invocations);
```

Comment:
The number of invocations changes since there is 1 more call now to establish that no data is left.